cv2/Error.pyi

```diff
@@ -0,0 +1,110 @@
+BadAlign = -21
+BAD_ALIGN = -21
+BadAlphaChannel = -18
```

Comment:
I wonder if this would be easier to maintain if it was in numerical rather than alphabetical order, like the OpenCV docs: https://docs.opencv.org/3.4/d1/d0d/namespacecv_1_1Error.html

Reply:
I've updated to what you get from running:
```py
from cv2 import Error
for key in dir(Error):
    print(f"{key} = {getattr(Error, key)}")
```

Reply:
That seems worse to me. It's harder to read and it also doesn't match the order in the docs.

How about sorting numerically (like the docs) and splitting it into two groups (ALL_CAPS and PascalCase) so it's easier for users to scan through? 

<details>
<summary>Like this...</summary>

```python
StsOk = 0
StsBackTrace = -1
StsError = -2
StsInternal = -3
StsNoMem = -4
StsBadArg = -5
StsBadFunc = -6
StsNoConv = -7
StsAutoTrace = -8
HeaderIsNull = -9
BadImageSize = -10
BadOffset = -11
BadDataPtr = -12
BadStep = -13
BadModelOrChSeq = -14
BadNumChannels = -15
BadNumChannel1U = -16
BadDepth = -17
BadAlphaChannel = -18
BadOrder = -19
BadOrigin = -20
BadAlign = -21
BadCallBack = -22
BadTileSize = -23
BadCOI = -24
BadROISize = -25
MaskIsTiled = -26
StsNullPtr = -27
StsVecLengthErr = -28
StsFilterStructContentErr = -29
StsKernelStructContentErr = -30
StsFilterOffsetErr = -31
StsBadSize = -201
StsDivByZero = -202
StsInplaceNotSupported = -203
StsObjectNotFound = -204
StsUnmatchedFormats = -205
StsBadFlag = -206
StsBadPoint = -207
StsBadMask = -208
StsUnmatchedSizes = -209
StsUnsupportedFormat = -210
StsOutOfRange = -211
StsParseError = -212
StsNotImplemented = -213
StsBadMemBlock = -214
StsAssert = -215
GpuNotSupported = -216
GpuApiCallError = -217
OpenGlNotSupported = -218
OpenGlApiCallError = -219
OpenCLApiCallError = -220
OpenCLDoubleNotSupported = -221
OpenCLInitError = -222
OpenCLNoAMDBlasFft = -223


STS_OK = 0
STS_BACK_TRACE = -1
STS_ERROR = -2
STS_INTERNAL = -3
STS_NO_MEM = -4
STS_BAD_ARG = -5
STS_BAD_FUNC = -6
STS_NO_CONV = -7
STS_AUTO_TRACE = -8
HEADER_IS_NULL = -9
BAD_IMAGE_SIZE = -10
BAD_OFFSET = -11
BAD_DATA_PTR = -12
BAD_STEP = -13
BAD_MODEL_OR_CH_SEQ = -14
BAD_NUM_CHANNELS = -15
BAD_NUM_CHANNEL1U = -16
BAD_DEPTH = -17
BAD_ALPHA_CHANNEL = -18
BAD_ORDER = -19
BAD_ORIGIN = -20
BAD_ALIGN = -21
BAD_CALL_BACK = -22
BAD_TILE_SIZE = -23
BAD_COI = -24
BAD_ROISIZE = -25
MASK_IS_TILED = -26
STS_NULL_PTR = -27
STS_VEC_LENGTH_ERR = -28
STS_FILTER_STRUCT_CONTENT_ERR = -29
STS_KERNEL_STRUCT_CONTENT_ERR = -30
STS_FILTER_OFFSET_ERR = -31
STS_BAD_SIZE = -201
STS_DIV_BY_ZERO = -202
STS_INPLACE_NOT_SUPPORTED = -203
STS_OBJECT_NOT_FOUND = -204
STS_UNMATCHED_FORMATS = -205
STS_BAD_FLAG = -206
STS_BAD_POINT = -207
STS_BAD_MASK = -208
STS_UNMATCHED_SIZES = -209
STS_UNSUPPORTED_FORMAT = -210
STS_OUT_OF_RANGE = -211
STS_PARSE_ERROR = -212
STS_NOT_IMPLEMENTED = -213
STS_BAD_MEM_BLOCK = -214
STS_ASSERT = -215
GPU_NOT_SUPPORTED = -216
GPU_API_CALL_ERROR = -217
OPEN_GL_NOT_SUPPORTED = -218
OPEN_GL_API_CALL_ERROR = -219
OPEN_CLAPI_CALL_ERROR = -220
OPEN_CLDOUBLE_NOT_SUPPORTED = -221
OPEN_CLINIT_ERROR = -222
OPEN_CLNO_AMDBLAS_FFT = -223
```

</details>

Reply:
I don't mind the order, as long as they're all present